### ISSUE TITLE
Display less days on the screen

### ISSUE THREAD

**Issue by Krands** [xsoh/Etar-Calendar#78]
Hi,

Nobody have a large screen and thus, display seven days on a small screen looks really cluttered. 

Is it possible to have a choice to display 3/4 days? This would be more handy.

Bests!

**Comment by jspricke** [comment]
Duplicate of #76

**Comment by Krands** [comment]
Hi, I am not issuing to display only work days ( 
Monday to Friday) but just some number of days (3-4) that way:
![tmp_27072-screenshot_2016-08-28-17-26-53-447608656](https://cloud.githubusercontent.com/assets/6728594/18034732/521a8f96-6d45-11e6-8c29-3983062dab66.png)

Because small screens display hardly the 7 days week...
It's technically possible satisfy both his issue and mine.

**Comment by jspricke** [comment]
oups, sorry. Reopening then.

**Comment by jpggithub** [comment]
+1
Sometimes, I put several events as taks to do at the same moment. I put my smartphone on landscape mode to read them. Not very friendly ;-)
It would be useful to be able to choose the number of days displayed.
Business calendar do it but it's not open source. One can use two fingers for dragging horizontally the or use a rule at the bottom of the screen to choose the number of days displayed (from 1 to 14 I remember).

**Comment by Gitsaibot** [comment]
If there are more people who want this feature I will try to implement it. But only after my other commits are merged.

**Comment by Gitsaibot** [comment]
On which view (Month/Week) days should be changeable ?

**Comment by Krands** [comment]
> On which view (Month/Week) days should be changeable ?

On week view?

**Comment by jpggithub** [comment]
Yes, on week view
